portal-backend/depmap/static/css/public/resources.scss

```diff
@@ -0,0 +1,12 @@
+.depmap-accordion {
+  margin: 0px !important;
+  .accordion-header {
+    padding: 8px 20px 8px 40px !important;
+    background-color: #f5f8fd !important;
+    border-bottom: 1px solid #585858 !important;
+    .accordion-title {
+      font-weight: 500;
+      font-size: 18px;
+    }
+  }
+}
```

Comment:
These styles override the Accordion component's `depmap-accordion` style. I feel like this could be somewhat simplified if the Accordion component took in style as props but I was unsure how this would affect all the other places in the portal that used it.